src/Caching/Hybrid/src/Internal/DefaultHybridCache.StampedeKey.cs

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.Extensions.Caching.Hybrid.Internal;
+
+partial class DefaultHybridCache
+{
+    internal readonly struct StampedeKey : IEquatable<StampedeKey>
```

Comment:
Developer experience question: do we want to sprinkle "debug view" attributes across types that we expect users will encounter as they debug code related to these paths?

Reply:
the key and flags are already exposed, and `ToString()` is implemented; should already be pretty transparent...?

Reply:
ToString seems adequate to me.